lib/trader.js

```diff
@@ -215,12 +356,16 @@ Trader.prototype.postTx = function postTx (tx) {
       }
 
       if (err.statusCode >= 500) throw err
-      if (isNetworkError(err)) return
+      if (this.isNetworkError(err)) return
       throw err
     })
 }
 
-function isNetworkError (err) {
+Trader.prototype.isNewServer = function isNewServer () {
+  return this.serverVersion && semver.gte(this.serverVersion, '8.1.0')
+}
```

Comment:
This name doesn't tell much, what does new server actually means? Can't we pass down the version as param and have a generic function to check for version?

Reply:
I did that initially, but it was unnecessarily verbose. It's always called with `this.serverVersion`, so I didn't see that as an improvement.

RE the name: it's a name like any other ^^ I can change it, no problem, but this method is temporary anyway, it won't be necessary after the old poller is removed. And that's what the method means: if the l-s has the GQL server.

Reply:
I mean passing down the required version string as a param, in this case it would be "8.1.0".

`isGraphqlPollEnabled` could be a good one if you don't make it generic.